src/test/java/teammates/test/cases/logic/InstructorsLogicTest.java

```diff
@@ -93,13 +94,16 @@ public void testAddInstructor() throws Exception {
         ______TS("failure: invalid parameter");
         
         instr.email = "invalidEmail.tmt";
-        
+        String expectedError =
```

Comment:
Ditto, ping @damithc, this asserts that ```instr.email``` is within the string, I find it ok and I need you to confirm too